### DISCUSSION TITLE
Update a Livewire component from a Controller

### DISCUSSION THREAD

**Discussion by hhhc** [livewire/livewire#4877] [1. Help]
Hi,

I have a Livewire component that hosts a map with some generic geo coordinates. The underlying data is updated regularly using a webhook (class XmlWebhookController extends Controller)  which updates the DB data.
Can I / how can I trigger the Livewire component to push an update to all users that currently display the map?

I saw this one https://github.com/livewire/livewire/discussions/4256 but in my case I have the update happening outside of a Livewire component...

Thanks for any advice.

**Comment by denizozturk** [comment]
You can do this with an event that is triggered after your database is updated.

Doc: https://laravel-livewire.com/docs/2.x/laravel-echo
